common/src/main/java/net/william278/husksync/database/Database.java

```diff
@@ -258,7 +257,9 @@ public final void pinSnapshot(@NotNull User user, @NotNull UUID versionUuid) {
     public enum Type {
         MYSQL("MySQL", "mysql"),
         MARIADB("MariaDB", "mariadb"),
-        MONGO("MongoDB", "mongo");
+        POSTGRES("PostgreSQL", "postgresql"),
+        MONGO("MongoDB", "mongo"),
+        MONGO_ATLAS("MongoDB", "atlas");
```

Comment:
```suggestion
        MONGO_ATLAS("MongoDB", "mongodb+srv");
```

Reply:
This has been removed in favour of a mong_settings configuration section